### PR TITLE
Upgrade 1000 volume

### PR DESCRIPTION
**Pull request by jyothis-gireesan**
### Description
API 1000 Support for Storage volume

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for go 1.11.x + gofmt checks .
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [X] New endpoints supported are updated in the endpoints-support.md file.
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/resource_volume.go [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+				Computed: true,
+			},
+			"device_specific_attributes": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"copy_state": {
+							Type:     schema.TypeString,
+							Computed: true,
```
Comment (madhav-bharadwaj) [inline oneview/resource_volume.go]: Are these attributes not set while update as well?

File: oneview/resource_volume.go [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"allocated_capacity": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"requesting_refresh": {
+				Type:     schema.TypeBool,
+				Computed: true,
```
Comment (madhav-bharadwaj) [inline oneview/resource_volume.go]: This attribute can be set while requesting for refresh, using a put call.

File: oneview/resource_volume.go (hunk 1 above)
Comment (jyothis-gireesan) [reply oneview/resource_volume.go]: its read only

File: oneview/resource_volume.go (hunk 2 above)
Comment (jyothis-gireesan) [reply oneview/resource_volume.go]: this is not for updating volume. that's why not given as an editable option.

**Pull request merged**